primitives/account/src/lib.rs

```diff
@@ -82,6 +82,12 @@ impl From<[u8; 32]> for AccountId20 {
 		Self(buffer)
 	}
 }
+impl From<sp_runtime::AccountId32> for AccountId20 {
+	fn from(account: sp_runtime::AccountId32) -> Self {
+		let bytes: &[u8; 32] = account.as_ref();
+		Self::from(*bytes)
+	}
+}
```

Comment:
What is this needed for? I wonder if this conversion is safe.

Reply:
It was because of this commit: https://github.com/paritytech/polkadot-sdk/commit/d1f678c0ec44e9734cf242247dab90c2678774a9

Where they added a bound to the benchmark
```rs
#[benchmarks(
	where
		<T as frame_system::Config>::AccountId: From<sp_runtime::AccountId32>,
		T::OffchainSignature: From<MultiSignature>,
)]
```

The code above is fine, in the lines above we already have:
```rs
impl From<[u8; 32]> for AccountId20 {
	fn from(bytes: [u8; 32]) -> Self {
		let mut buffer = [0u8; 20];
		buffer.copy_from_slice(&bytes[..20]);
		Self(buffer)
	}
}
```